tests/json/mod.rs

```diff
@@ -2,6 +2,7 @@ use std::str;
 use std::collections::BTreeMap;
 
 use cookie_factory::*;
+use cookie_factory::lib::std::io::Write;
```

Comment:
I guess the remaining question here is, if we want to re-export the io compat module for no-std environments in a nicer way or if this is fine.

Reply:
Well, let's call it "ok" for now